### PR TITLE
Refactor/core dependencies

### PR DESCRIPTION
**Pull request by Kaskyi**
Move login service.
```ts
import { LoginService } from '@xm-ngx/core/auth';
import { LoginService } from '@xm-ngx/components/login';
```

### CODE REVIEW

File: packages/core/theme/src/stores/xm-theme-store.service.ts [hunk 1]
```diff
@@ -12,6 +12,12 @@ import { XmTheme } from '../interfaces/xm.theme';
  */
 export class XmThemeStore {
     public get(): XmTheme | null {
+
+        // TODO:TEMPORARY
+        //  While user theme store not supported
+        //  the main way to set a theme is provide it via ui config
+        return null;
```
Comment (sergeysenja1992) [inline packages/core/theme/src/stores/xm-theme-store.service.ts:19]: what reason to do it?

File: packages/core/user/src/account.service.ts [hunk 2]
```diff
@@ -45,11 +44,11 @@ export class AccountService {
         return this.http.post(SERVER_API_URL + ACCOUNT_TFA_DISABLE_URL, {}, {observe: 'response'});
     }
 
-    public getProfile(req?: any): Observable<XmEntity> {
+    public getProfile(req?: any): Observable<any> {
```
Comment (sergeysenja1992) [inline packages/core/user/src/account.service.ts:47]: this method must be typed, why we make code worse?

File: packages/entity/src/entity-list-card/entity-list-card-options.model.ts [hunk 3]
```diff
@@ -64,8 +47,8 @@ export interface ActionsOptions extends ActionOptions {
     navigateByInnerUrl?: string;
     functionKey?: string;
     privilege?: string[] | string;
-    handler?: (entity: XmEntity) => any;
-    actionCondition?: (entity: XmEntity) => any;
+    handler?: (entity: IId) => any;
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-list-card/entity-list-card-options.model.ts:50]: this is not equals replacement

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xm-webapp",
-  "version": "4.0.64",
+  "version": "4.0.65",
```
Comment (sergeysenja1992) [inline package.json:3]: pls increment second number
it's not a minor changes, it`s breaking changes

File: packages/core/user/src/account.service.ts (hunk 2 above)
Comment (Kaskyi) [reply packages/core/user/src/account.service.ts:47]: first i want to split /core package with other packages, complete packages build, than refactor and provide property dependencies between packages.

File: package.json (hunk 4 above)
Comment (Kaskyi) [reply package.json]: done

**Pull request merged**